docs/architecture.md

```
@@ -0,0 +1,98 @@
+2. Registers `register_activation_hook()` → `LSX_WETU_Importer::register_activation_hook()`.
+3. Requires `classes/class-lsx-wetu-importer.php`, which at the bottom of the file instantiates the plugin's single global instance:
+
+   ```php
+   $lsx_wetu_importer = LSX_WETU_Importer::get_instance();
+   ```
+
+   That single line triggers the entire hook-registration cascade for the whole plugin.
+
+## Class map
+
+```
```

markdownlint-cli2 (0.23.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)